### PR TITLE
Feature/in537 your org

### PR DESCRIPTION
**Pull request by GlenHani**

### CODE REVIEW

**Review by benleolander — DISMISSED** [review]

**Review by BEISKRAM — DISMISSED** [review]

File: INZFS.MVC/Controllers/FundApplicationController.cs [hunk 1]
```diff
@@ -438,8 +449,8 @@ public async Task<IActionResult> Save([Bind(Prefix = "submit.Publish")] string s
             else
             {
                 await _session.CancelAsync();
-                currentPage = _applicationDefinition.Application.AllPages.FirstOrDefault(p => p.Name.ToLower().Equals(pageName));
-                return PopulateViewModel(currentPage, model);
+                var _currentPage = _applicationDefinition.Application.AllPages.FirstOrDefault(p => p.Name.ToLower().Equals(pageName));
+                return PopulateViewModel(_currentPage, model);
             }
```
[the remark this answers is not in the document]
Comment (GlenHani) [reply INZFS.MVC/Controllers/FundApplicationController.cs:454]: issue fixed

File: INZFS.MVC/Views/FundApplication/CurrencyInput.cshtml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    @if (Model.ShowMarkAsComplete)
+    {
+        @Html.Partial("_MarkAsComplete", Model)
+    }
+
+
+
+
+    <div class="edit-item-secondary group">
```
[the remark this answers is not in the document]
Comment (GlenHani) [reply INZFS.MVC/Views/FundApplication/CurrencyInput.cshtml]: resloved

File: INZFS.MVC/Views/FundApplication/SingleRadioSelectInput.cshtml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+<span class="govuk-caption-l">Question @Model.QuestionNumber of @Model.TotalQuestions</span>
+
+
+<form asp-action="save" asp-route-pagename="@Model.PageName" method="post" enctype="multipart/form-data">
+
+    <input type="hidden" name="PageName" value="@Model.PageName" />
+    <input type="hidden" name="Kind" value="@nameof(RadioSingleSelectModel)" />
+
+    @if (!ViewData.ModelState.IsValid)
+    {
```
[the remark this answers is not in the document]
Comment (GlenHani) [reply INZFS.MVC/Views/FundApplication/SingleRadioSelectInput.cshtml]: Resloved

File: INZFS.MVC/Views/FundApplication/SingleRadioSelectInput.cshtml [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+
+
+    @Html.Partial("_Accordion", "")
+    @if (Model.ShowMarkAsComplete)
+    {
+        @Html.Partial("_MarkAsComplete", Model)
+    }
+
+    <div class="edit-item-secondary group">
```
[the remark this answers is not in the document]
Comment (GlenHani) [reply INZFS.MVC/Views/FundApplication/SingleRadioSelectInput.cshtml]: Resolved

**Review by BEISKRAM — APPROVED** [review]

**Review by sunilbeis — APPROVED** [review]

**Pull request merged**